test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1286,20 +1288,24 @@ class select_ep_transport : public test_ucp_wireup {
         modify_config("NUM_EPS", num_eps1);
         entity *e1 = create_entity();
 
-        if (!has_resource(*e1, "dc_mlx5") && !has_resource(*e1, "rc_mlx5") &&
-            !has_resource(*e1, "rc_verbs")) {
-            UCS_TEST_SKIP_R("RMA transports are not present");
-        }
-
         /* Second context initialized with num_eps below threshold */
         modify_config("NUM_EPS", num_eps2);
         entity *e2 = create_entity();
 
         e1->connect(e2, get_ep_params());
         e2->connect(e1, get_ep_params());
 
+        auto tl1 = rma_transport(e1);
+        auto tl2 = rma_transport(e2);
+
+        ASSERT_EQ(!tl1, !tl2);
+        if (tl1 == NULL) {
+            ASSERT_TRUE(ucs::is_aws());
+            UCS_TEST_SKIP_R("RMA transports are not present");
+        }
```

Comment:
why is previous check not enough?

Reply:
not sure which check you are referring to, but it is only on aws that we can have no rma transport, as IB/EFA does not have it yet.

Reply:
sorry, i thought the below check was in the original code
```
 if (!has_resource(*e1, "dc_mlx5") && !has_resource(*e1, "rc_mlx5") &&
            !has_resource(*e1, "rc_verbs")) {
            UCS_TEST_SKIP_R("RMA transports are not present");
 }
```